sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkDoubleCounter.java

```diff
@@ -97,41 +97,44 @@ public void unbind() {
     }
   }
 
-  static final class Builder extends AbstractInstrumentBuilder<SdkDoubleCounter.Builder>
-      implements DoubleCounterBuilder {
+  static final class SdkDoubleCounterBuilder
+      extends AbstractInstrumentBuilder<SdkDoubleCounterBuilder> implements DoubleCounterBuilder {
 
-    Builder(
+    SdkDoubleCounterBuilder(
```

Comment:
Giving these builders more descriptive names gives `AbstractInstrumentBuilder#toString()` a useful classname to include.